BE.BL/Clients/ClientAggregateFactory.cs

```diff
@@ -28,26 +29,34 @@ public ClientAggregate Create(CreateClientCommand createClientCommand)
 
         public ClientAggregate Create(long clientId)
```

Comment:
The contents of this whole method should be in the `ClientsWriteRepository.GetClientAggregate` method. That method should return a fully built `Client` class. You can get rid of this method after you move that code.